velox/docs/functions/presto/window.rst

```diff
@@ -130,19 +136,31 @@ Returns the value at the specified offset from the beginning of the window. Offs
 can be any scalar expression. If the offset is null or greater than the number of values in the window, null is
 returned. It is an error for the offset to be zero or negative.
 
-.. function:: lag(x[, offset [, default_value]]) -> [same as input]
+.. function:: lag(x[, offset[, default_value]]) -> [same as input]
+
+Returns the value at ``offset`` rows before the current row in the window partition.
+Offsets start at ``0``, which is the current row. The default ``offset`` is ``1``.
+The offset can be a constant value or a column reference. If the offset is ``null``, ``null`` is
+returned. If the offset refers to a row that is not within the partition, the
+``default_value`` is returned, or if ``default_value`` is not specified ``null``
+is returned.
+
+If ``IGNORE NULLS`` is specified, ``null`` values are ignored during offset counting.
```

Comment:
Given that we have a paragraph at the top describing Ignore NULLs, the paragraph seems not needed any more?